extensions/sql-database-projects/src/common/iconHelper.ts

```diff
@@ -16,6 +16,7 @@ export class IconPathHelper {
 	public static databaseProject: IconPath;
 	public static colorfulSqlProject: IconPath;
 	public static sqlEdgeProject: IconPath;
+	public static sqlDbProject: IconPath;
```

Comment:
add "Azure" in the name so it's clear this is the Azure sql db icon

Reply:
![image](https://user-images.githubusercontent.com/17187348/160208314-782f1140-111f-4533-8f9e-2142d7582892.png)